examples/record-contract/record-contract.ncl

```diff
@@ -11,7 +11,7 @@ let PortElt
   | doc "A contract for a port element of a Kubernetes configuration"
   = {
     name | Str,
-    containerPort | #Port,
+    container_port | #Port,
```

Comment:
Illustrating the previous comment, the field names in this example are taken from actual Terraform schema, so we don't have the freedom of picking the case.

Reply:
The official reference is https://kubernetes.io/docs/reference/kubernetes-api/. Another useful and better tailored resource is the original example it was inspired from (IIRC): https://github.com/kubernetes/examples/blob/master/guestbook-go/guestbook-controller.json.